eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/publisher/email/EagleMailClient.java

```diff
@@ -54,6 +54,8 @@ public EagleMailClient(final Properties config) {
             velocityEngine = new VelocityEngine();
             velocityEngine.setProperty(RuntimeConstants.RESOURCE_LOADER, "classpath");
             velocityEngine.setProperty("classpath.resource.loader.class", ClasspathResourceLoader.class.getName());
+            velocityEngine.setProperty(RuntimeConstants.RUNTIME_LOG_LOGSYSTEM_CLASS,org.apache.velocity.runtime.log.NullLogChute.class.getName());

```

Comment:
I think it would be better to put the fully qualified name in the `import` snippet. In addition, here is a problem that `EagleMailClient` need be refactoring for sloving code duplication, so, in addition to` alert-engine` module, there `eagle-common` module is also need to be modified.

```java
import org.apache.velocity.runtime.log.NullLogChute;

velocityEngine.setProperty(RuntimeConstants.RUNTIME_LOG_LOGSYSTEM_CLASS,org.apache.velocity.runtime.log.NullLogChute.class.getName());
```